_posts/en/meetings/2016-06-02-meeting.md

```diff
@@ -0,0 +1,130 @@
+
+## Notes / short topics
+
+There aren't a lot of people interested in doing the softfork backports for 0.11 and there isn't a lot of user interest for it either. Releasing without good review/testing might be more dangerous than not doing a release at all. The software life cycle page still promises to maintain the previous version.
+
+## Segwit review
+ 
+### background
+ 
+Developers are working on a soft fork to introduce segregated witness onto Bitcoin mainnet. Segregated witness (segwit) allows transaction signature data to be stored outside of the data hashed to produce transaction identifiers, removing all known forms of third-party malleability, allowing full nodes to compile the current UTXO set without downloading all signatures, and laying the groundwork for fraud proofs that can allow lightweight (SPV) clients to help enforce more of the consensus rules. The segwit soft fork also allows miners to substitute 1 byte of block space with 4 bytes of segwit data, increasing transaction capacity for wallets that use segwit. segregated witness BIPs: [BIP141][], [BIP142][], [BIP143][], [BIP144][] and [BIP145][]
+
+### meeting comments
+ 
+Sipa's planning to make the BIP9/GBT changes, remove segnet and the positive witness flag and then create a parallel rebase with a clear history, but which leads to the same tree. The positive witness flag was a flag to indicate you want to serialize the witnesses. Since you almost always want to serialize the witnesses outside of a few exceptions it's better to have a negative witness flag. Another reason is that a failure of setting the positive flag will not usually be detected, but a failure to set a negative flag will. Having both flags is also an option, but would result in more code changes shattered all over the place.
+
+Since you don't want users to have segwit addresses before the roll-out, the new wallet code can be introduced behind a hidden configuration option that tests could use.
+
+jl2012 brought up the issue that our witness program definition is limited to 16 versions, which is not easily extended without introducing a new witness storage. The easy solution to allow witness programs to be slightly larger is a hard fork change compared to the current code, which would result in a testnet fork, since segwit is already activated there. Since the worst thing that can happen is a reindex for testnet nodes and 16 is too few sipa will change the version limit.
+
+### meeting conclusion
+
+- Review [#7749][] (Enforce expected outbound services) and [#8083][] (Add support for dnsseeds with option to filter by servicebits)
+- Extend the max witness program length
+
+## Compactblock testing
+### background
+ 
+[BIP152][]: “Compact block relay” is a proposed idea by BlueMatt for decreasing the bandwidth used during block relay by using short transaction IDs for transactions that should be in the mempool of the node. As a side-effect this also results in reducing the block transfer latency.
```

Comment:
AFAIK the current code being tested won't beat the Fast Relay Network.  

It's primarily targeted at bandwidth reduction, and as a side-effect reduces latency over the p2p network.

The followup version with block coding(mentioned below) may beat the relay network though.